Fix link text for anchor links to GDS Way pages (#1024)
alphagov/govuk-prototype-kit: docs/documentation/setting-up-git.md

```
The message you put in the speech marks should be descriptive of the changes you are committing. This will help in the future if you or someone else needs to look back at your changes and know why you made them.

> More information on writing good commit messages is in the [Git style guide in the 'How to store source code' page of the GDS Way](https://gds-way.cloudapps.digital/standards/source-code.html#commit-messages).

Reviewer: ```suggestion
Read more about writing good commit messages in the [Git style guide in the 'How to store source code' page of the GDS Way](https://gds-way.cloudapps.digital/standards/source-code.html#commit-messages).
```
